Content.Server/Nutrition/Components/DrinkComponent.cs

```diff
@@ -26,6 +26,10 @@ public sealed class DrinkComponent : Component
         [DataField("isOpen")]
         internal bool DefaultToOpened;
 
+        [ViewVariables]
+        [DataField("isOpenable")]
+        public bool IsOpenable = true;
```

Comment:
Currently this bool is really only preventing the open-text from being shown when examining.

The actual drink is still open-able by interactions (and needs to be open in order to be drunk). So this variable probably needs a more accurate name.

Reply:
Are cans (beer, cola, etc.) only drinks that are openable?

Reply:
No, only glasses are currently not openable, every other drink that uses the drink base is openable. At least that’s how it should be.